lib/facter/pe_status_check.rb

```diff
@@ -203,6 +203,22 @@
     { S0022: validity }
   end
 
+  chunk(:S0024) do
+    next unless PEStatusCheck.primary? || PEStatusCheck.replica? || PEStatusCheck.postgres?
+
+    # Check discard directory. Newest file should not be less than a week old. Recent files indicate an issue that causes PuppetDB to reject incoming data.
+    newestfile = Dir.glob('/opt/puppetlabs/server/data/puppetdb/stockpile/discard/*.*').max_by { |f| File.mtime(f) }
```

Comment:
Can we match the logic be consistent with https://github.com/puppetlabs/puppetlabs-pe_status_check/blob/main/lib/facter/pe_status_check.rb#L118-L122?